ui/apps/platform/src/services/ComplianceScanConfigurationService.ts

```diff
@@ -79,6 +79,15 @@ export type ComplianceScanConfigurationStatus = {
     lastExecutedTime: string | null; // either ISO 8601 date string or null when scan is in progress
 };
 
+// @TODO: This may change and be moved around depending on how backend implements it.
```

Comment:
> depending on how backend implements it

You might suggest Luis ask David S. what principles he used to decide when to create a new proto file:
https://github.com/stackrox/stackrox/tree/master/proto/api/v2